app/controllers/solidus_klarna_payments/api/sessions_controller.rb

```diff
@@ -3,6 +3,8 @@
 module SolidusKlarnaPayments
   module Api
     class SessionsController < ::Spree::Api::BaseController
+      skip_before_action :verify_authenticity_token, only: :create
```

Comment:
Verify_authenticity_token is not executed by Spree::Api::BaseController. I don't think this will fix the problem.